docs/self-managed/installation-methods/helm/cloud-providers/amazon/amazon-eks/terraform-setup.md

```diff
@@ -718,6 +720,86 @@ Running these commands will clean up both the job and the secret, ensuring that
 </TabItem>
 </Tabs>
 
+### Accessing internal infrastructure
+
+:::warning Not recommended in production
+
+These approaches are intended for **development and troubleshooting purposes only**.
+For a production cluster, you should use a proper VPN or other secure access methods.
+
+:::
+
+Some infrastructure components (like OpenSearch dashboards or Aurora PostgreSQL databases) are only accessible from inside the EKS network.
+You can use a temporary pod as a **“jumphost”** to tunnel traffic to these components.
```

Comment:
```suggestion
You can use a temporary pod as a **“jump host”** to tunnel traffic to these components.
```

while both work, I found `jump host` more commonly in e.g. Wikipedia or Tailscale usage.